projects/ad777x_ardz/de10nano/system_qsys.tcl

```diff
@@ -0,0 +1,5 @@
+
+source $ad_hdl_dir/projects/common/de10nano/de10nano_system_qsys.tcl
+source ../common/ad777x_ardz_qsys.tcl
+set_instance_parameter_value sys_spi {clockPolarity} {0}
+
```

Comment:
After update, the de10nano doesn't build. You need to add something similar with:
https://github.com/analogdevicesinc/hdl/blob/master/projects/cn0540/de10nano/system_qsys.tcl#L8 
to configure the SYS_ID properly

Reply:
Solved and tested in V5.